src/js/media-controller.js

```diff
@@ -710,10 +739,95 @@ class MediaController extends MediaContainer {
     els.splice(index, 1);
   }
 
-  /**
-   * Media Controller should not mimic the HTMLMediaElement API.
-   * @see https://github.com/muxinc/media-chrome/pull/182#issuecomment-1067370339
-   */
+  keyboardShortcutHandler(e) {
+    if (this.contains(e.target) || this.shadowRoot.contains(e.target)) {
```

Comment:
question(non-blocking): Do we need these checks?

Reply:
Maybe not, since I guess the event bubbling should guarantee that it's coming from a child element?

Reply:
yup that's what I was thinking. We may also get into some weirdness with `contains` checks for nested web component use cases.